### DISCUSSION TITLE
2022 年终总结 - forecho's Blog

### DISCUSSION THREAD

**Discussion by giscus** [forecho/hugo-blog#29] [Announcements]
# 2022 年终总结 - forecho's Blog

引言
又到了一年一度的年终总结了，从今年开始我准备按照阳历写总结了（往年是按照农历春节），我想早点儿结束这操蛋的 2022 年。
回顾 2022
生活
今年真是魔幻的一年，前两年疫情对我们的生活还影响不大，从今年年初就开始各种「骚操作」，4 月上海封城了一个月，而深圳我们机会核酸没断过，365 天，至少核酸了 250 天，每天为了核酸都要浪费至少半个小时时间。
好在年末的时候终于不再动态清零了，感谢那些为我们「呐喊」的人。
去年搬到小区来住，没想到最后选的这个房子，隔音真差，在卧室都能听到楼上走路的声音，而且楼上的人经常晚上 11 点多才睡觉，早上 7 点多才起床，真是让人无法忍受。
今年年底老婆怀孕了，我终于要升级为爸爸了，这算是今年唯一的好消息了。🎉

https://blog.forecho.com/review-of-2022.html

**Comment by hutusi** [comment]
厉害厉害👍
